docs/modules/clients/pages/java.adoc

```diff
@@ -24,14 +66,72 @@ which you may already have done to start using Hazelcast:
 </dependency>
 ----
 
-You can find Hazelcast Java client's code samples https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients[here^].
+--
+====
+
+You can find {java-client} code samples in the https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients[Hazelcast Code Samples repository].
+
+TIP: For a tutorial on getting started with Java in an embedded topology, see xref:getting-started:get-started-java.adoc[].
+
+=== Get started with {java-client-new} (BETA)
+
+To get started using the {java-client-new}, you need to include the `hazelcast-java-client-{page-latest-supported-java-client-new}.jar` dependency in your classpath. You can then start using this client as if
+you are using the Hazelcast API. 
```

Comment:
@ihsandemir you want to change this line?

Reply:
done